frontend/class-breadcrumbs.php

```diff
@@ -255,14 +255,15 @@ private function find_deepest_term( $terms ) {
 			$parents = $this->get_term_parents( $term );
 
 			if ( count( $parents ) >= $parents_count ) {
-				$parents_count = count( $parents );
 
 				// If higher count.
 				if ( count( $parents ) > $parents_count ) {
```

Comment:
I've discussed this with @moorscode and we concluded that it is beter to remove this if-statement. There is no bug reported, so this can be taken is intended behaviour. If we fix this 'potential bug', users might experience another behaviour.

Reply:
@andizer Makes sense to me. Shall I close this PR and open a new one or just replace the commit in this PR with a new one ?

Reply:
Good question. I would make a new commit in this PR, just to have a history.

Reply:
Will do.

Personally I see history as only relevant when something has gone into trunk. What has not been in trunk, is a try-out/proposal and subject to change until accepted (merged with a proper commit message)